build_odin.sh

```diff
@@ -77,9 +77,9 @@ Linux)
 	LDFLAGS="$LDFLAGS -Wl,-rpath=\$ORIGIN"
 	;;
 OpenBSD)
-	CXXFLAGS="$CXXFLAGS $($LLVM_CONFIG --cxxflags --ldflags)"
-	LDFLAGS="$LDFLAGS -liconv"
+	CXXFLAGS="$CXXFLAGS -I/usr/local/include $($LLVM_CONFIG --cxxflags --ldflags)"
 	LDFLAGS="$LDFLAGS $($LLVM_CONFIG --libs core native --system-libs)"
+	LDFLAGS="$LDFLAGS -L/usr/local/lib -liconv"
```

Comment:
From what I can read online, this `/usr/local/` directory is supposed to be configurable with `$PREFIX` environment variable by the user. Some users redefine it to `/opt`. Maybe setting this path to `$PREFIX` instead of hardcoding it as `/usr/local` is better?

Reply:
Also it seems like FreeBSD has similar situation, though I don't know whether distribution-provided compilers include the search paths for `$PREFIX`. Maybe worth adding the same changes to FreeBSD, a broken search path wouldn't hurt

Reply:
True. I would like to use `pkg-config` but the libiconv port doesn't have an associated `*.pc` in the search directory:
```
openbsd$ ls /usr/lib/pkgconfig/       
expat.pc      fuse.pc       libcrypto.pc  libelf.pc     libssl.pc     openssl.pc    sndio.pc      zlib.pc
openbsd$ ls /usr/local/lib/pkgconfig/
```
Checking for `$PREFIX` makes sense. Users are likely not building ports themselves, so by hardcoding my initial thought was to offload the complexity to ports hackers who are accustomed to carrying patches.